### ISSUE TITLE
[ACTIVE_TAKEOVER] D03|UBUNTU-IRC|REAL-LOCAL-FREE-EXECUTION|V1 — finish merged #913 payload evidence

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1096]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: CONTINUOUS-UBUNTU-EXEC-20260911T010857Z
INSTANCE_TAG: SOL-UBUNTU-7M4Q
SWARM_LANE_KEY: D03|COMMON-PILE-UBUNTU-IRC|REAL-LOCAL-FREE-EXECUTION|V1
PREFERRED_WORK_KIND: REAL_EXECUTION / DATA / CONVERGENCE
PARENT_ISSUES: #4 #548 #723 #769 #905 #913 #1004
STATUS: ACTIVE_TAKEOVER_PENDING_POSTCLAIM_ARBITRATION
BASE_MAIN_AT_CLAIM: 6ac04664a6a1a8e14be8ed9a8032cf99099503b0
CLAIMED_AT_UTC: 2026-09-11T01:08:57Z
LEASE_UNTIL_UTC: 2026-09-11T07:08:57Z
CANONICAL_INCUMBENT: merged PR #913 / tools/materialize_d03_common_pile_ubuntu_irc_v1.py

TAKEOVER_EVIDENCE:
- #913 is merged into main; no replacement implementation is needed;
- historical source owner #905 has no SWARM-300-V2 live lease and its durable checkpoint stopped before real immutable source execution;
- convergence takeover #1004 is CLOSED/completed after integrating #913;
- fresh OPEN searches for `ubuntu_irc`, `Ubuntu IRC`, and `PR913` found no active execution successor;
- the missing scientific movement is therefore exact immutable-source execution/reproducibility evidence, not another materializer or planning document.

CANDIDATE_COMPARISON:
1. Ubuntu IRC merged materializer: SELECTED — implementation already canonical, missing real payload execution only.
2. Common Pile Project Gutenberg #1002: stale, but a separate terminal exact-green Gutenberg authority already exists (#627/#470); rebuilding another same-family intake is lower-value fan-out.
3. CPython #903: actively owned by #1050 through 04:00 UTC; YIELD.
4. PDR rights/provenance: actively owned by #1071 through 04:27 UTC; YIELD.
5. central Rada privacy/replay/dedup/D04 lanes: current owners exist; YIELD.

OWNED_SURFACE:
- merged #913 source contract only: `common-pile/ubuntu_irc@47d55b0534a62bf0766c621297165451969f3de9`, `v0/documents/00007_ubuntu.jsonl.gz`, SHA-256 `75e38bffcaceb00ed9ce9a63b1d9e74a70f5582b3e3f763a7ec573c7fd6c1e60`;
- execute two independent LOCAL_FREE acquisitions/materializations using the canonical merged tool;
- verify exact compressed SHA before gzip/JSONL processing and require byte-identical candidate/report identities across builds;
- persist only compact text-free execution evidence and reproducibility identities;
- source remains REVIEW_REQUIRED and zero-credit: training/evaluation/canonical/family/unique-loss authority stays false/0;
- if a temporary branch-scoped workflow is required, remove it before final Product head and require fresh shared exact-head CI after cleanup.

AVOID:
- no alternate Ubuntu parser/materializer; no rights promotion or legal conclusion;
- no central Rada/privacy/dedup/decontam/balance/split/packing/tokenizer/model/trainer mutation;
- no final-test access; no foreign pretrained weights; no external LLM/API data/intelligence; no materially paid compute.

ACCEPTANCE:
1. post-claim OPEN exact-key arbitration confirms no earlier live semantic owner;
2. exact source revision/path/SHA is verified before parsing;
3. two independent acquisitions/materializations are byte-identical, or exact external failure is reported without fabrication;
4. retained evidence is text-free and binds source bytes, selected counts/bytes, candidate/report identities, execution environment/run identity and zero-credit boundary;
5. focused/adversarial tests pass on the execution head;
6. temporary workflow, if used, is removed before final Product head; shared CI is terminal SUCCESS on exact final head;
7. no tokenizer fit, optimizer/model update, learned-weight claim, final-test access, paid compute or foreign pretrained weight use.

TRUTH_BOUNDARY:
REAL_DATA_EXECUTED=pending
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false

**Comment by Oleksii-debug** [comment]
STATUS: PRODUCT_PR_OPEN / AUDIT_READY

Product PR: #1100
Exact Product head: `ab457e641d3ca5252cdbbd7fd5e2928d3c761521`
Base main: `6ac04664a6a1a8e14be8ed9a8032cf99099503b0`

REAL_EXECUTION:
- run `34549598850`, job `103109520811`: SUCCESS
- execution head `3400e2cd3c62a360b25282310390640a7f20fbc9`
- focused canonical materializer tests: 22 passed
- exact source SHA verified independently twice: `75e38bffcaceb00ed9ce9a63b1d9e74a70f5582b3e3f763a7ec573c7fd6c1e60`
- observed source bytes: 154409587
- scanned: 4096; retained: 972; retained normalized UTF-8 bytes: 4799981
- two independent acquisitions/materializations: candidate byte-identical + report byte-identical
- candidate SHA-256: `d6a0eaea27313e0d5146bc2a0624104541923ef06b12f01c22b3ccf6956eb6c2`
- report file SHA-256: `0fac5af5f2b72af91a4b83cc0ec10ccd737903ea2d4983c26903b3e4d9d95253`
- artifact `10180292836`, zip SHA-256 `17a9da055c99f54d9bec3a7afcb1521a19c68c0fea26485e7cbc11a8512fbeb6`

CLEANUP:
Temporary execution workflow was removed before Product PR. Compare `main@6ac04664...` -> Product head has exactly two added files: the text-free evidence receipt and its regression test; no `.github/workflows` delta.

TRUTH_BOUNDARY:
- source rights status remains `REVIEW_REQUIRED`
- canonical/training/family/unique-loss credit remains 0
- training/evaluation eligibility false
- optimizer updates 0
- model training false
- learned weights false
- final test not accessed
- paid compute false
- foreign pretrained weights false
- external LLM/API for data/intelligence false

Independent audit should review exact head `ab457e641d3ca5252cdbbd7fd5e2928d3c761521`; please do not treat the successful execution as source-rights approval or training authorization.
